xml/System.Threading.Channels/ChannelOptions.xml

```diff
@@ -33,7 +33,7 @@
       </AssemblyInfo>
       <Parameters />
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Protected constructor that initializes an instance of the <see cref="T:System.Threading.Channels.ChannelOptions" /> abstract class. This class cannot be instantiated.</summary>
```

Comment:
Why does this say "This class cannot be instantiated" when describing the ctor?

Reply:
I see why you ask. It might not be necessary in the constructor. Besides, it should describe the type, but someone noticing `abstract` should know it cannot be instantiated. Perhaps mentioning that this constructor cannot be called because it is `protected` would be more appropriate, but again, the protected keyword already shows up in Docs, so it's extra information that is not necessary.
I'll simplify the text to this:

```suggestion
        <summary>Initializes an instance of the <see cref="T:System.Threading.Channels.ChannelOptions" /> abstract class.</summary>
```